docs/modules/ROOT/pages/tests-configuration.adoc

```diff
@@ -2,18 +2,21 @@
 
 == `configuration.properties`
 
-The Vividus configuration includes the following parts: profiles, environments and suite.
+The Vividus configuration includes the following parts: profiles, environments and suites.
 
-=== Suite
+=== Suites
 
-The suite is defined by the property:
-
-```properties
-configuration.suite
-```
+The property `configuration.suites` defines the suites set. It is a comma separated set of suite file addresses:
 
+.configuration.properties
+----
+configuration.suites=testSuite,anotherDir/anotherTestSuite
+----
 The Vividus suite consists of the unlimited number of batches. In its turn the batch consists of any number of stories represented by the files with `*.story` extension. The batches are run sequentially, while the stories in the batch are run in parallel.
 
+Each property in suite files should be unique among suite files provided if you suppose to use them in suites set.
+Otherwise, the behaviour can be unpredictable.
```

Comment:
I propose to replace this statement with the following one:
```
The suites are loaded one by one starting from the last one in the sequence, in other words, the values of the properties defined in the first suite will take precedence over the values of the same properties defined in all subsequent suites.
```

Reply:
Done.